example-new-site/design4-post.html

```
@@ -0,0 +1,96 @@
+    <p style="font-size:19px;line-height:1.75;margin:0 0 22px">Seeing Botwick&rsquo;s network inspired me to build one of my own.</p>
+
+    <p style="font-size:19px;line-height:1.75;margin:0 0 22px">I processed roughly 600 Granola meeting transcripts with Claude Code and turned them into an Obsidian knowledge graph.</p>
+
+    <figure style="margin:12px 0 26px">
+      <x-import component-from-global-scope="image-slot" from="../image-slot.js" id="qp-fig" shape="rounded" radius="5" placeholder="Drop one frame" src="https://harper.blog/2026/03/11/2026-immaculate-knowledge-graph/network_hu_60cd7bb93f09d12a.png" style="width:100%;height:320px;display:block" hint-size="640px,320px"></x-import>
+      <figcaption style="font-size:13px;color:var(--pmut,#8C877E);margin-top:10px">Everyone and everything I know!</figcaption>
+    </figure>
+
+    <blockquote style="margin:0 0 24px;padding:0 0 0 20px;border-left:3px solid var(--pline,#ECE9E3);font-size:20px;line-height:1.5;font-style:italic;color:var(--pmut,#8C877E)">Laziness is key. Just make it work for you.</blockquote>
+
+    <pre style="font-family:ui-monospace,Menlo,monospace;font-size:14px;line-height:1.7;background:var(--pbg2,#F7F5F1);border:1px solid var(--pline,#ECE9E3);border-radius:6px;padding:18px 20px;margin:0 0 24px;overflow:auto">harper@magic [†] ~/ > muesli sync</pre>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)